### PR TITLE
/ui: add `ScreenGauge`

### LINKED ISSUE #187: /ui: add `ScreenGauge`
Port `ScreenGauge` from https://github.com/nestauk/eurito_indicators_ui/pull/12/files by @saabi

### PR DESCRIPTION
**Pull request by mindrones**
Closes #187

Port `ScreenGauge` from nestauk/eurito_indicators_ui#12 by @saabi

### CODE REVIEW

File: packages/components/ui/src/gauges/screen/README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+<script>
+	import {screen} from '@svizzle/ui/src/gauges/ScreenGauge.svelte';
+
+	console.log($screen);
+</script>
+
+{#if $screen.size.medium && $screen.orientation.landscape}
+	<p>Medium (landscape)</p>
+{/if}
+/>
```
Comment (saabi) [inline packages/components/ui/src/gauges/screen/README.md:98]: What do you think about expanding the example to include some class selectors?

Comment (saabi) [reply packages/components/ui/src/gauges/screen/README.md:98]: Maybe adding that they allow for progressive enhancements starting from `small` as default

File: packages/components/ui/src/gauges/screen/ScreenGauge.svelte [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+	export const screen = writable();
+</script>
+
+<script>
+	import * as _ from 'lamb';
+	import {getTruthyValuesKeys, joinWithBlank} from '@svizzle/utils';
+
+	const getClasses = _.pipe([getTruthyValuesKeys, joinWithBlank]);
+	const sampleText = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz';
```
Comment (saabi) [inline packages/components/ui/src/gauges/screen/ScreenGauge.svelte]: Maybe export `sampleText`  for allowing users to provide their own text if (i.e. using different alphabets)

Comment (mindrones) [reply packages/components/ui/src/gauges/screen/ScreenGauge.svelte]: nice idea, ok

**Review by saabi — APPROVED** [review]

**Pull request merged**